src/deepsparse/transformers/pipelines_cli.py

```diff
@@ -241,8 +243,8 @@ def response_to_json(response: Any):
         return [response_to_json(val) for val in response]
     elif isinstance(response, dict):
         return {key: response_to_json(val) for key, val in response.items()}
-    elif hasattr(response, "json") and callable(response.json):
-        return response.json()
+    elif isinstance(response, BaseModel):
+        return dict(response)
```

Comment:
Calling dict here cause response.json() returns a string, and we need a object for well formed output with `json.dump(...)`